.github/workflows/validate_pr.yml

```diff
@@ -23,12 +23,18 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+      - run: |
+          # create the master branch
+          git branch ${{ github.base_ref }} origin/${{ github.base_ref }}
+          # create the pr branch
+          git branch ${{ github.head_ref }} origin/${{ github.head_ref }}
```

Comment:
Why do you need this after using `fetch-depth: 0` ?

Reply:
fetch does not create the branches, and to no messup the links generated, I can't just use `origin/master`

Reply:
You don't need the branches, you just need the commits they're referring to be available when running `git log` etc.

Reply:
yes but this needs branches to make a proper link here:
https://github.com/lumapps/changelog-generator/blob/rewrite_release_note/changelog_generator/changelog_template.jinja#L1-L2